src/components/colorbar/attributes.js

```diff
@@ -228,4 +228,10 @@ module.exports = overrideAll({
             ].join(' ')
         }
     },
+    _deprecated: {
```

Comment:
How about dropping the `deprecation` here?

Reply:
Hmm.. I see then it would collide with the the title which is declared as an object.

Reply:
@archmoj That sounds fine but I'm unsure about the `attributes` syntax for accepting both an object (with specific nested keys) and a string... this is the attribute definition right now for `title` in colorbar:

```
    title: {
        text: {
            valType: 'string',
            description: 'Sets the title of the color bar.'
        },
        font: fontAttrs({
            description: 'Sets this color bar\'s title font.'
        }),
        side: {
            valType: 'enumerated',
            values: ['right', 'top', 'bottom'],
            description: [
                'Determines the location of color bar\'s title',
                'with respect to the color bar.',
                'Defaults to *top* when `orientation` if *v* and ',
                'defaults to *right* when `orientation` if *h*.',
            ].join(' ')
        }
```

My point is that there doesn't seem to be a place to define `valType` for `title` if we are also defining nested attributes under `title`. Do you have a suggestion?

Reply:
Let's see what @alexcjohnson suggest here.

Reply:
Or perhaps we should just handle this String option on the plotly.py side?

Reply:
@archmoj Do you have an example of an `extra` option being used elsewhere in the codebase?

Reply:
Something like this: https://github.com/plotly/plotly.js/pull/6990/files#r1591049384
Instead of `extra` you may add another option to coerce and name it `stringOk` or something else.